assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -36,6 +36,9 @@ export const TakeActionBoxoutEditor = ({
     stickyOnMobile,
   } = attributes;
 
+  const {options: p4_options} = window.p4_vars;
+  const isNewIA = p4_options.new_ia === 'on' && true;
```

Comment:
Why `&& true`?

Reply:
It is another approach to define `boolean`. Basically, if the condition is `true`, then the value is `thruty`, then is `falsy`.

If the new IA is enabled
![Screen Shot 2024-11-26 at 11 07 27](https://github.com/user-attachments/assets/b8fc11d7-3687-4583-8336-887ccb1715a9)

If the new IA is disabled
![Screen Shot 2024-11-26 at 11 07 56](https://github.com/user-attachments/assets/d73618df-2a97-4f92-bc83-0f398ac51aae)

Reply:
But if you just do `p4_options.new_ia === 'on'` doesn't it return the same?